pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -1928,6 +1928,16 @@ SELECT length(gen_random_uuid()::BYTES), gen_random_uuid() = gen_random_uuid()
 ----
 16 false
 
+query IB
+SELECT length(gen_random_ulid()::BYTES), gen_random_ulid() = gen_random_ulid()
+----
+16 false
+
+query TTTTTT
+SELECT uuid_ulid_to_string('0178951c-b665-30a7-19a2-a18999834858')
+----
+01F2AHSDK562KHK8N1H6CR6J2R
```

Comment:
I think we should test roundtripping this back to a UUID, we can add another builtin so we can go both ways for the test.

Reply:
Makes sense, I'll add `ulid_to_uuid` here based on the previous comment.